### ISSUE TITLE
Hummingbot Client - Freezes on main screen after password prompt

### ISSUE THREAD

**Issue by glenntomas** [CoinAlpha/hummingbot#3042]
**Describe the bug**
Users are reporting that when opening hummingbot client it freezes on the main screen and works after 5 minutes

**Steps To Reproduce**
1. Open hummingbot client on the binary build, docker build, and source build 
2. Enter the password, then click ok
3. After entering the main screen it freezes at 2 seconds duration 

**Screenshots**
![image](https://user-images.githubusercontent.com/73885708/110328265-19017200-8056-11eb-9e24-fea85eb7153b.png)

**Release version**
v0.36.0

**Attachments**
```
2021-03-08 21:18:45,230 - 16592 - hummingbot.core.utils.async_utils - ERROR - Unhandled error in background task: 
Traceback (most recent call last):
  File "hummingbot\core\utils\async_call_scheduler.py", line 80, in _coro_scheduler
asyncio.exceptions.CancelledError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "hummingbot\core\utils\async_utils.py", line 9, in safe_wrapper
  File "hummingbot\core\utils\async_call_scheduler.py", line 128, in call_async
  File "hummingbot\core\utils\async_call_scheduler.py", line 117, in schedule_async_call
  File "hummingbot\core\utils\async_call_scheduler.py", line 80, in _coro_scheduler
  File "site-packages\async_timeout\__init__.py", line 55, in __aexit__
  File "site-packages\async_timeout\__init__.py", line 92, in _do_exit
asyncio.exceptions.TimeoutError
```

**Discord Username:**
Rahxephonroh#4225
Crispried#9814
Peteronfire#8360
MarcoDePolo#0890
kaiofabian 🇧🇷#8967

**Comment by marcodepolo** [comment]
hello, I'm not too sure what happened but suddenly my bot window is all buggy and not responding. if I resize the terminal its not resizing, no echo when I type something on the keyboard... I tried restarting from scratch but that didnt work out. I even restarted the laptop but no luck. any idea whats wrong ?
![Screenshot from 2021-03-08 14-34-33](https://user-images.githubusercontent.com/13634941/110329631-2e0ddf00-801d-11eb-9cbd-3a41970dda9b.png)
[logs_hummingbot.log](https://github.com/CoinAlpha/hummingbot/files/6101753/logs_hummingbot.log)

**Comment by scope-li** [comment]
Same here. 

Hummingbot (0.36.0) on AWS with Docker.
Existing Instances and also new Instances have the same problem.

**Comment by glenntomas** [comment]
In tomorrow's release, we will fix the issue where the bot hangs after entering password. 

In the meantime, here's how to fix the issue:
- Open the `conf_global.yml` file in the `conf` directory (or the `hummingbot_conf` directory in `hummingbot_files` if Docker install)
- Find the parameter `ethereum_token_list_url`
- Change it to : `ethereum_token_list_url: https://defi.cmc.eth`
- Save the file
- Restart the bot and it shouldn't hang

**Comment by dennisocana** [comment]
Fixed in version 0.37
